crypto/src/main/java/org/springframework/security/crypto/encrypt/Encryptors.java

```diff
@@ -90,6 +90,21 @@ public static TextEncryptor text(CharSequence password, CharSequence salt) {
 		return new HexEncodingTextEncryptor(standard(password, salt));
 	}
 
+	/**
+	 * Creates an encryptor for queryable bytes that uses standard password-based
+	 * encryption. Uses a 16-byte all-zero initialization vector so encrypting the same
+	 * data results in the same encryption result. This is done to allow encrypted data to
+	 * be queried against.
+	 *
+	 * @param password the password used to generate the encryptor's secret key; should
+	 * not be shared
+	 * @param salt a hex-encoded, random, site-global salt value to use to generate the
+	 * secret key
+	 */
+	public static BytesEncryptor queryable(CharSequence password, CharSequence salt) {
```

Comment:
Why is the method called 'queryable'?, Should this class allow encryptions specific to make query.

Reply:
I'm not sure if it's the best name. But currently we have:
- (Encryptor) which is "stronger"
- (Encryptor) which is "standard"
- (Encryptor) which is "delux"
- so an Encryptor which is "queryable" seemed not so bad.

Currently we have a method named "queryableText", which returns a `TextEncryptor`. Maybe the same method which returns a `BytesEncryptor` should be called "queryableBytes"? But we don't have "strongerBytes" or "standardBytes", just "stronger" and "standard", so maybe the "queryable" is enough?